src/Raven.Server/Documents/BlittableMetadataModifier.cs

```diff
@@ -366,6 +367,23 @@ private unsafe bool AboutToReadPropertyNameInMetadataUnlikely(IJsonParser reader
                     }
 
                     goto case -1;
+                case 21: //Raven-Expiration-Date
```

Comment:
Didn't you just rename this property?

Reply:
This is for 3.5 -> 4.0 smuggler import

Reply:
Okay, best to add a comment to explain it

Reply:
This whole class is doing a lot of 3.5 -> 4.0 migrations and IIRC was created for this purpose.